piet-svg/src/lib.rs

```diff
@@ -42,13 +48,30 @@ impl RenderContext {
         }
     }
 
+    /// Set the size that the SVG should render to
+    pub fn set_size(&mut self, size: Size) {
```

Comment:
is it intended/helpful for this to be mutated in the middle of an operation, or should it be set on  create and subsequently be immutable? This would better match the semantics of other backends.

Reply:
So the reason I did it like this is because (unlike the other backends), SVG images have a concept of "unsized" - you don't have to set the size or viewBox of an SVG image.

It might be that we want to restrict this to forcing a size (like other backends). In this case I could change as you suggested.

Reply:
What is the expected behaviour when an svg is unsized?

I'm wondering if this is compatible with the overall design of the crate. For instance, If an SVG is unsized, what should be the behaviour of the `clear` fn?

Reply:
I vaguely remember another reason for wanting the ability to set size after creation but I can't remember it now. I will make sure all my things work without the `set_size` method, then I propose we get rid of it and make the api like the other backends.

Reply:
still waiting on this, yea? otherwise I think I'm happy to merge :)

Reply:
done :)